packages/openneuro-components/src/tooltip/Tooltip.tsx

```diff
@@ -5,9 +5,15 @@ export interface TooltipProps {
   tooltip: string
   flow: 'up' | 'down' | 'left' | 'right'
   children: React.ReactNode
+  className: string
```

Comment:
This will fix the CI issue
```suggestion
  className?: string
```